README.md

```diff
@@ -29,11 +34,14 @@ This contract is a singleton contract. Once deployed, to be able to use this con
 
 ## Usage
 
-In order to configure this contract to work with a Gnosis Safe, the following steps should be taken:
+In order to configure this contract to work with a Gnosis Safe, use the following steps:
 
-1. Set the guard for the Gnosis Safe to the address of the Time Restriction contract.
-2. Call the `addTimeRange` function with the days and time ranges in which transactions will be allowed.
-3. Now the Gnosis Safe will only allow transactions to be executed within the time ranges specified.
+1. Deploy a timelock contract with the desired delay, guardian, pause duration, whitelisted targets, calldatas, with the users' Gnosis Safe as the owner.
+2. Queue a transaction in the Gnosis Safe to perform the following actions:
```

Comment:
Would be nice to have a singleton deployer contract like [this](https://github.com/wilsoncusack/safe-singleton-deployer-sol/blob/main/src/SafeSingletonDeployer.sol)